include/common.hpp

```diff
@@ -5,7 +5,9 @@
 #include <type_traits>
 #include <sstream>
 #include <iomanip>
+
 #include "version.hpp"
+#include "greentea_libdnn_config.h"
```

Comment:
@naibaf7 When I install greentea this file it's not found. What do you suggest to do with this file during installation?